ga4gh/backend.py

```diff
@@ -192,6 +192,64 @@ def _getEnd(cls, variant):
         return variant.end
 
 
+class VariantAnnotationsIntervalIterator(IntervalIterator):
+    """
+    An interval iterator for annotations
+    """
+
+    def __init__(self, request, parentContainer):
+        super(VariantAnnotationsIntervalIterator, self).__init__(
+            request, parentContainer)
+        self._featureIds = set(self._request.feature_ids)
```

Comment:
It's my impression that the request may not contain feature_ids. Is some type of none-check or exception handling needed here to get the desired functionality?

        try:
          self._featureIds = set(self._request.feature_ids)
        except TypeError:
          self._featureIds = []